### PR TITLE
wallet-sdk cli

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: packages/cli/src/commands/dids.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    console.log(dids);
+  });
+
+didsCommand
+  .command('create')
+  .description('List wallet dids')
+  .action(async () => {
+    const wallet: IWallet = await getWallet();
+
+    // implement did creation logic, start with did key
```
Comment (mike-parkhill) [inline packages/cli/src/commands/dids.ts:26]: are you tracking all these improvement thoughts anywhere else? I feel like these comments might get lost

**Review by mike-parkhill — APPROVED** [review]
I think this will be very handy for dev and test

**Comment by maycon-mello** [comment]
> I think this will be very handy for dev and test

Yeah, I was planning to publish it as a built-in package (all the deps included in one single file). So it should have zero setup time.

It should work like `npx @docknetwork/wallet-sdk-cli credentials list`. Only requirement is to have npm and nodejs

File: packages/cli/src/commands/dids.ts (hunk 1 above)
Comment (maycon-mello) [reply packages/cli/src/commands/dids.ts:26]: Yes. I pushed this as placeholder for only. But I'm currently working on migrating the basic verification flow to the wallet-sdk. DID creation is a requirements on it. And I will leave wallet-backups (import, export), accounts, and transactions for the future.

**Pull request merged**